environmentmodel/environment-model_ontology.ttl

```diff
@@ -1,17 +1,16 @@
 @prefix dcterms: <http://purl.org/dc/terms/> .
-@prefix environment-model: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environment-model/> .
+@prefix environmentmodel: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/environmentmodel/> .
 @prefix gax-core: <https://w3id.org/gaia-x/core#> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-environment-model: a owl:Ontology ;
-    rdfs:label "ontology definition for environment-model"@en ;
+environmentmodel: a owl:Ontology ;
+    rdfs:label "ontology definition for environmentModel"@en ;
     dcterms:contributor "Mirco Nierenz (TG)" ;
     owl:versionInfo "0.1"^^xsd:float .
 
-environment-model:Asset a owl:Class ;
-    rdfs:label "environment-model" ;
-    rdfs:comment "attributes for 3d environment models"@en ;
+environmentmodel:environmentModel a owl:Class ;
```

Comment:
Classnames PascalCase, attributes camelCase
-->EnvironmentModel would be correct.

Reply:
I adapted that.